src/axom/quest/examples/quest_marching_cubes_example.cpp

```diff
@@ -176,18 +204,20 @@ struct Input
     ndim = std::max(ndim, fcnCenter.size());
     ndim = std::max(ndim, inPlane.size());
     ndim = std::max(ndim, perpDir.size());
+    ndim = std::max(ndim, gyroidScale.size());
```

Comment:
Minor: Would this work?
```suggestion
    ndim = std::max({ndim, fcnCenter.size(), inPlane.size(), perpDir.size(), gyroidScale.size() });
```

(All the entities might need to have the same type to use the initializer list -- https://en.cppreference.com/w/cpp/algorithm/max )

Reply:
Thanks!